Tests/Models/BrickDimensionTests.swift

```diff
@@ -174,6 +174,56 @@ class BrickDimensionTests: XCTestCase {
                 compact: .auto(estimate: .fixed(size: 50))
             )))
     }
+    
+    func testBrickRangeDimention() {
+        setupScreen(isPortrait: true, horizontalSizeClass: .compact, verticalSizeClass: .regular)
+        
+        let additionalRangePairs: [RangeDimensionPair] = [(dimension: .ratio(ratio: 0.25), minimumLength: CGFloat(700)),
+                                                          (dimension: .ratio(ratio: 0.5), minimumLength: CGFloat(350))] // Intentionally out of order
+        let regularDimensionRange : BrickDimension = .dimensionRange(range: BrickRangeDimention(minimum: .ratio(ratio: 1.0), additionalRangePairs: additionalRangePairs))
+        XCTAssertEqual(regularDimensionRange.value(for: 320, startingAt: 0), 320)
+        setupScreen(isPortrait: true, horizontalSizeClass: .regular, verticalSizeClass: .regular)
+        XCTAssertEqual(regularDimensionRange.value(for: 350, startingAt: 0), 175)
+        XCTAssertEqual(regularDimensionRange.value(for: 768, startingAt: 0), 192)
+        XCTAssertEqual(regularDimensionRange.dimension(withValue: nil).value(for: 400, startingAt: 0), 400)
+    }
+    
+    func testVariableDimensionEquality() {
+        let additionalRangePairs1: [RangeDimensionPair] = [(dimension: .ratio(ratio: 1/2), minimumLength: CGFloat(350))]
+        let dimensionRange1: BrickDimension = .dimensionRange(range: BrickRangeDimention(minimum: .ratio(ratio: 1.0), additionalRangePairs: additionalRangePairs1))
+        let additionalRangePairs2: [RangeDimensionPair] = [(dimension: .ratio(ratio: 0.5), minimumLength: CGFloat(350))]
+        let dimensionRange2: BrickDimension = .dimensionRange(range: BrickRangeDimention(minimum: .ratio(ratio: 0.5 + 0.5), additionalRangePairs: additionalRangePairs2))
```

Comment:
Not a fan of calling enum .dimensionRange and then initializing an object that is BrickRangeDimension, provided comments on how to adjust this call so that it's similar to the other BrickDimension cases we have